src/cb_admin/pages/Visitor.js

```diff
@@ -139,9 +144,20 @@ export default class VisitorProfile extends React.Component {
   };
 
   onClickResend = () => {
+    this.setState({ resendQrCodeState: resendQrCodeState.PENDING });
```

Comment:
Are you planning on using this value?

Reply:
It has a UX advantage. If you click resend again it removes the error message.